README.md

```diff
@@ -275,20 +619,68 @@ We can discuss what is automated, what is not, and how that can be improved. Mor
 what are the production services and software offered by HPC, and can we trace their entire pipeline
 from source code to production deployment? Is there room for improvement?
 
+> there is a lot of discussion aroudn software stacks like this right now -- we should probably
+> sync up on it. There are more reasons than just RSE-ops to have a common software stack for HPC
+> at this level.  It also helps us push on vendors.
+
 ## Summary of Opportunities
 
 The following areas of RSE-ops are not well developed, and we have opportunity to build or seek out tools to fill the space.
 
 - **Application Development**: Development framework for interactive (browser based) applications for HPC. This relies on being able to ensure secure access in a browser.
+
+> I think people are calling this "cloud services" or "web applications". If you say "app
+> development in HPC people think simulation.
+
 - **Automated Scaling**:  Being able to scale a job, task, or service without thinking about it.
 - **On Demand Resources**: An HPC user should theoretically be able to request a custom resource, and "throw it away" when done. For HPC an administrative team would still need to monitor and maintain the resources, but to the user there could be a more interactive, customized experience. E.g., "I want an instance with this amount of memory, storage, and with this software installed. And I want to customize and request it in a web interface."
+
+> confused about this one because this is what SLURM and Flux do. You can ask for these things, and
+> you are given a temporary allocation. The allocation is then used by someone else when you are
+> done. Allocation is bare metal, not instance-based, but this is what RMs do. The web interface
+> part wuld be new but people are generally managing complex resource requests for ensembles. I
+> think resource requests are more like configuration-as-code than they are like ewb forms -- at
+> least for big apps and studies.
+
 - **Continuous Integration**: Application developers on HPC rarely test their code from version control directly on systems where they are intended to be run. It should be possible to better connect where code lives to where it will ultimately be used to run tests and builds.
+
+> this isn't true -- many times they *only* test it where it is intended to run, and running it
```

Comment:
This is 100% true for places that are not national labs. Ask any academic center researcher how they run their tests from GitHub on their cluster and you'll get a bunch of blank stares, and at best someone will mention Jenkins they saw somewhere.

Reply:
right... so it's not 100% true.

Reply:
That’s why I said “rarely” :) I guess I don’t have a percentage, but it seems to be less common to be able to do that.

Reply:
There are also academic teams who test only on the clusters they want to run on -- not necessarily automated tests though.  I've seen people do cronjobs, run stuff manually -- lots of not so great ways to run tests.

I just think we have to be careful not to make broad generalizations.  The point here is that we need to make it easy to run tests on the clusters.  I would just say that it can be very difficult to do that -- not make a generalization that is likely not true for a number of teams.

Reply:
I agree, let me try to reword this.